lib/foods.js

```diff
@@ -1,28 +1,56 @@
+const local = 'http://localhost:3000/api/foods'
```

Comment:
I've been testing locally, and then changing it to production for my PRs (and after testing it on prod, obviously). I'm worried this could be easily forgotten.